src/app/settings/admin/bucket-settings/destinations/component.ts

```diff
@@ -77,6 +77,13 @@ export class DestinationsComponent implements OnInit {
     return !!destination?.credentials;
   }
 
+  isDefault(name: string): boolean {
+    return (
+      this.seed?.spec?.etcdBackupRestore?.defaultDestination &&
+      this.seed.spec.etcdBackupRestore.defaultDestination === name
```

Comment:
`this.seed?.spec?.etcdBackupRestore?.defaultDestination === name`.

Reply:
right. fixed 👍